atomic_reactor/cli/task.py

```diff
@@ -10,7 +10,9 @@
 from atomic_reactor.tasks.clone import CloneTask
 from atomic_reactor.tasks.common import TaskParams
 from atomic_reactor.tasks.orchestrator import OrchestratorTask, OrchestratorTaskParams
-from atomic_reactor.tasks.sources import SourceBuildTask, SourceBuildTaskParams
+from atomic_reactor.tasks.sources import (
+    SourceBuildExitTask, SourceBuildPhasesTask, SourceBuildTaskParams
```

Comment:
Again for consistency with binary tasks, maybe the classes should be `SourceExitTask`, `SourceBuildTask`, `SourceTaskParams`?